styles.css

```diff
@@ -46,16 +49,25 @@ nav > div{
 	border: 2px solid #777674;
 }
 
-.navSmallScreen{
-	display: none;
+
+.navigation > a{
+	text-decoration: none;
+}
+
+.navigation > a:visited{
+	color: #000000;
+}
+
+.navigation > a:link{
+	color: #000000;
 }
 
 
 
-/* title and about me section*/
+/* title section*/
 header{
 	display: flex;
-	/*height: 800px;*/
+	height: 700px;
 	justify-content: center;
 	align-items: center;
 	padding: 180px 0;
```

Comment:
Turning padding off in the inspector doesn't change anything in my browser. This is the same for the 'background-repeat' declaration. I think that when you use background-size: cover in this instance, it means you don't have to use 'no-repeat'.

Reply:
Thanks!
No repeat has been removed but changing the height has an affect in browser.  I think it only has an affect if you're screen is very wide like mine.